9c-internal/9c-stake-test/values.yaml

```diff
@@ -0,0 +1,173 @@
+# Default values for 9c-internal cluster.
+# This is a YAML-formatted file.
+
+
+clusterName: "9c-internal-v2"
+logLevel: "debug"
+
+global:
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    tag: "git-e414be95d4640cf0520780333860b8af7090d2a6"
+
+  appProtocolVersion: "1/54684Ac4ee5B933e72144C4968BEa26056880d71/MEUCIQDFvvGTtUSSRj524xJf0EcxmmU6UaXVDAfp3fsb7P6fkwIgdSicMlCnEUZwsPr+W6VLg+2ReY4+FIRC2vB1Yssz7EU=/ZHU5OnRpbWVzdGFtcHUxMDoyMDIzLTA1LTE5ZQ=="
+  genesisBlockPath: "https://9c-dx.s3.ap-northeast-2.amazonaws.com/stake-test-genesis-block"
+  trustedAppProtocolVersionSigner: "02529a61b9002ba8f21c858224234af971e962cac9bd7e6b365e71e125c6463478"
+  peerStrings:
+  - "033369e95dbfd970dd9a7b4df31dcf5004d7cfd63289d26cc42bbdd01e25675b6f,9c-internal-stake-validator.nine-chronicles.com,31234"
+  iceServers:
+  - "turn://0ed3e48007413e7c2e638f13ddd75ad272c6c507e081bd76a75e4b7adc86c9af:0apejou+ycZFfwtREeXFKdfLj2gCclKzz5ZJ49Cmy6I=@turn-us.planetarium.dev:3478"
+
+  networkType: Internal
+  consensusType: pbft
+
+  resetSnapshot: false
+  useExternalSecret: true
+
+snapshot:
+  downloadSnapshot: false
+  slackChannel: "9c-internal"
+  image: "planetariumhq/ninechronicles-snapshot:git-dc31bc2b777d9ca18d101bd1ab81351cce1b32c4"
+  path: internal
+
+# if you want to delete PVC with the volume provisioned together, set this value "Delete"
+volumeReclaimPolicy: "Retain"
+
+seed:
+  count: 0
```

Comment:
is it okay without any seed node?

Reply:
I thought it'll be ok cause it starts from empty genesis block.